### PR TITLE
iptraf-ng: 1.1.4 -> 1.2.1

### PR DESCRIPTION
**Pull request by devhell**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/iptraf-ng/default.nix [hunk 1]
```diff
@@ -1,24 +1,25 @@
-{ stdenv, fetchurl, ncurses }:
+{ stdenv, fetchFromGitHub, ncurses }:
 
 stdenv.mkDerivation rec {
-  version = "1.1.4";
+  version = "v1.2.1";
```
Comment (fabaff) [inline pkgs/applications/networking/iptraf-ng/default.nix:4]: The version should be 

```suggestion
  version = "1.2.1";
```

See

File: pkgs/applications/networking/iptraf-ng/default.nix [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
   pname = "iptraf-ng";
 
-  src = fetchurl {
-    url = "https://fedorahosted.org/releases/i/p/iptraf-ng/${pname}-${version}.tar.gz";
-    sha256 = "02gb8z9h2s6s1ybyikywz7jgb1mafdx88hijfasv3khcgkq0q53r";
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = pname;
+    rev = version;
```
Comment (fabaff) [inline pkgs/applications/networking/iptraf-ng/default.nix:10]:
```suggestion
    rev = "v${version}";
```

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 107793` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>iptraf-ng</li>
  </ul>
</details>

File: pkgs/applications/networking/iptraf-ng/default.nix [hunk 3]
```diff
@@ -1,24 +1,25 @@
...
+    sha256 = "1f91w1bjaayr6ld95z2q55ny983bb0m05k1jrw2bcddvcihaiqb1";
   };
 
   buildInputs = [ ncurses ];
 
-  configurePhase = ''
-    ./configure --prefix=$out/usr --sysconfdir=$out/etc \
-                --localstatedir=$out/var --sbindir=$out/bin
+  installPhase = ''
+    make DESTDIR="$out" prefix=/usr sbindir=/bin sysconfdir=/etc localstatedir=/var install
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/iptraf-ng/default.nix:17]: I would prefer if we used makeFlags here.

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107793` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>iptraf-ng</li>
  </ul>
</details>

`gpl2` is a deprecated license, check if project uses `gpl2Plus` or `gpl2Only` and change `meta.license` accordingly.

File: pkgs/applications/networking/iptraf-ng/default.nix (hunk 3 above)
Comment (devhell) [reply pkgs/applications/networking/iptraf-ng/default.nix]: Thank you @SuperSandro2000, good call. I had totally forgotten about them. Done!

File: pkgs/applications/networking/iptraf-ng/default.nix (hunk 2 above)
Comment (devhell) [reply pkgs/applications/networking/iptraf-ng/default.nix]: Thank you @fabaff, totally right, I have no idea what I was thinking :rofl:

**Review by stigtsp — COMMENTED** [review]
Seems like files end up in `$out/homeless-shelter/`

```
[nix-shell:~/.cache/nixpkgs-review/pr-107793]$ find results/iptraf-ng/
results/iptraf-ng/
results/iptraf-ng/homeless-shelter
results/iptraf-ng/homeless-shelter/share
results/iptraf-ng/homeless-shelter/share/man
results/iptraf-ng/homeless-shelter/share/man/man8
results/iptraf-ng/homeless-shelter/share/man/man8/iptraf-ng.8
results/iptraf-ng/homeless-shelter/sbin
results/iptraf-ng/homeless-shelter/sbin/iptraf-ng

```

**Comment by devhell** [comment]
> Seems like files end up in `$out/homeless-shelter/`
> 
> ```
> [nix-shell:~/.cache/nixpkgs-review/pr-107793]$ find results/iptraf-ng/
> results/iptraf-ng/
> results/iptraf-ng/homeless-shelter
> results/iptraf-ng/homeless-shelter/share
> results/iptraf-ng/homeless-shelter/share/man
> results/iptraf-ng/homeless-shelter/share/man/man8
> results/iptraf-ng/homeless-shelter/share/man/man8/iptraf-ng.8
> results/iptraf-ng/homeless-shelter/sbin
> results/iptraf-ng/homeless-shelter/sbin/iptraf-ng
> ```

Sorry, I missed that.

**Review by stigtsp — APPROVED** [review]
Tested OK on linux x64

New src url looks legit, same used in [Arch](https://archlinux.org/packages/extra/x86_64/iptraf-ng/)

**Pull request merged**